chapters/install/os/via-pip.rst

```diff
@@ -51,14 +45,14 @@ For Docker users the following command updates Jina in the Docker container:
    docker pull jinaai/jina
 
 
-Install from the master branch
+Install Jina development or pre-release
```

Comment:
I think this might introduce confusion with our wording for `pip install jina[devel]` which sounds similar but installs something different.

What's wrong with the old heading?

Reply:
In theory nothing, but installing from the master branch is our version of development or nightly build. So I just tried to align with those terms..

Reply:
What about we stick to `jina development version`? 
```suggestion
Install Jina development via pip
```

Reply:
Plus, I'm a bit confused with the head level of this section. It should be part of the `installation via pip` rather than `Upgrade Jina`